src/main/kotlin/com/openlattice/assembler/AssemblerConnectionManager.kt

```diff
@@ -140,25 +140,43 @@ class AssemblerConnectionManager(
                 }
             }
 
-            organization.members
-                    .filter { it.id != "openlatticeRole" && it.id != "admin" }
-                    .filter {
-                        securePrincipalsManager.principalExists(it)
-                    } //There are some bad principals in the member list some how-- probably from testing.
-                    .forEach { principal ->
-                        configureUserInDatabase(
-                                datasource,
-                                buildOrganizationDatabaseName(organizationId),
-                                buildPostgresUsername(securePrincipalsManager.getPrincipal(principal.id))
-                        )
-                    }
+            addMembersToOrganization(dbname, datasource, organization.members)
 
             createForeignServer(datasource)
 //                materializePropertyTypes(datasource)
 //                materialzieEntityTypes(datasource)
         }
     }
 
+    fun addMembersToOrganization(dbName: String, dataSource: HikariDataSource, members: Set<Principal>) {
+        members.filter { it.id != "openlatticeRole" && it.id != "admin" }
```

Comment:
do we have constants for these strings somewhere?

Reply:
I only see admin in SystemRole

Reply:
Will move openLatticeRole to SystemRole too